### PR TITLE
Updates docs to be aware of India Data Residency

### PR DESCRIPTION
**Pull request by austinpray-mixpanel**
Updates docs with addition of India-specific endpoints, updates to the privacy documentation, and instructions for configuring SDKs for India Data Residency.

- make cspell aware of DPDPA
- Updates random parts of docs that reference api-eu to also reference api-in
- streamline privacy page to be aware of both india and EU
- Adds /docs/privacy/in-residency with detailed instructions on how to use India DC
- Updates all SDKs to be aware of India DC
- Updates debugging page with india instructions

### CODE REVIEW

**Comment by myronkaifung** [comment]
Hi @austinpray-mixpanel , I just pushed some other changes that updated the file format of the SDK pages from MD to MDX. Mind if I add the reference to India Residency in the SDK pages in a separate PR so we can resolve the conflict?

**Comment by austinpray-mixpanel** [comment]
> Hi @austinpray-mixpanel , I just pushed some other changes that updated the file format of the SDK pages from MD to MDX. Mind if I add the reference to India Residency in the SDK pages in a separate PR so we can resolve the conflict?

@myronkaifung yep that sounds good to me

**Review by myronkaifung — APPROVED** [review]
thanks!

File: pages/docs/tracking-methods/sdks/python.mdx [hunk 1]
```diff
@@ -442,6 +442,21 @@ mp_eu = Mixpanel(
...
+### India Data Residency
+
+Route data to Mixpanel's India servers by using a custom `Consumer` with an `api_host` set to the India domain.
+
+**Example Usage**
+```py
+from mixpanel import Mixpanel
+
+# use custom consumer with URLs set to Mixpanel's India domain
+mp_eu = Mixpanel(
```
Comment (austinpray-mixpanel) [inline pages/docs/tracking-methods/sdks/python.mdx]:
```suggestion
mp_in = Mixpanel(
```

**Pull request merged**